koans/01--AboutArrays.js

```diff
@@ -7,10 +7,10 @@ describe("About Arrays", function() {
   it("should create arrays", function() {
     var emptyArray = [];
 
-    expect(typeof(emptyArray)).to.equal(FILL_ME_IN);
+    expect(typeof(emptyArray)).to.equal('object');
     // A mistake? - http://javascript.crockford.com/remedial.html
 
-    expect(emptyArray.length).to.equal(FILL_ME_IN);
+    expect(emptyArray.length).to.equal(0);
```

Comment:
Hey here's an example....

    // The length of an empty array _must_ be 0, because it's empty.
    // The `length` property is the count of the number of items in the array.